errors/errors.go

```diff
@@ -9,4 +9,5 @@ var (
 	ErrImageEmpty                   = errors.New("image is empty")
 	ErrCannotResolvePolicyException = errors.New("cannot resolve policy exception")
 	ErrCannotInitializeChecks       = errors.New("unable to initialize checks")
+	ErrCannotListChecks             = errors.New("unable to list checks")
```

Comment:
since we don't wrap errors from `Run` this can be removed.

Reply:
fixed